epoxy-adapter/build.gradle

```diff
@@ -1,26 +1,35 @@
 apply plugin: 'com.android.library'
+apply from: "$rootDir/blessedDeps.gradle"
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion "24.0.2"
+  compileSdkVersion rootProject.COMPILE_SDK_VERSION
+  buildToolsVersion rootProject.ANDROID_BUILD_TOOLS_VERSION
 
   defaultConfig {
-    minSdkVersion 16
-    targetSdkVersion 24
+    minSdkVersion rootProject.MIN_SDK_VERSION
+    targetSdkVersion rootProject.TARGET_SDK_VERSION
   }
 }
 
+def deps = rootProject.blessedDeps;
```

Comment:
Instead of using `rootProject.blessedDeps` as the global reference should we just make it `deps` so we don't have to do `def deps = rootProject.blessedDeps;` in every module?